irma-css/src/components/irma-form.scss

```diff
@@ -72,6 +70,59 @@
   border: 1px solid $border-color;
   border-radius: $border-radius;
 
+  .irma-web-content {
+    @include reset;
+    @include center;
+    flex: 1 1 auto;
+    margin: 0 1rem 1rem 1rem;
+
+    background-color: $content-bg-color;
+    border-radius: $border-radius * 0.6;
+
+    .irma-web-centered {
+      @include reset;
+      height: auto;
+      text-align: center;
+
+      img,
+      canvas {
+        cursor: none;
+      }
+
+      // Make sure margin-top of first child and margin-bottom of last child
+      // are equal to prevent issues while centering.
+      > p {
+        &:first-child {
+          margin-top: 0;
+        }
+        &:last-child {
+          margin-bottom: 0;
+        }
+      }
```

Comment:
I had to move this part inside `.irma-web-centered` to prevent the no-descending-specificity from complaining. It is meant to prevent the margins to differ and messing up the centering. It is therefore only useful inside .irma-web-centered.